Classes/NodeProxyGui2.sc

```diff
@@ -395,12 +403,19 @@ NodeProxyGui2 {
 			var layout, paramVal;
 			var slider, valueBox, textField, staticText;
 
+			paramVal = nodeProxy.get(key);
+
 			layout = HLayout.new(
-				[StaticText.new().string_(key), s: 1],
+				if (paramVal.isArray && {paramVal.every(_.isNumber)}) {
```

Comment:
i'd like to see ```and:``` here instead of ```&&```.  that'd follow the coding style in the rest of the class and also ```and:``` is a tiny bit faster.